pkg/runtime/session.go

```diff
@@ -13,12 +13,17 @@
 
 package runtime
 
-import "github.com/aws/aws-sdk-go/aws/session"
+import (
+	ackv1alpha1 "github.com/aws/aws-controllers-k8s/apis/core/v1alpha1"
 
-func NewSession() (*session.Session, error) {
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+)
+
+func NewSession(region ackv1alpha1.AWSRegion) (*session.Session, error) {
 	// NOTE(jaypipes): session.NewSession() is needed for the STS::AssumeRole
 	// stuff we will need to do...
-	sess, err := session.NewSession()
+	sess, err := session.NewSession(aws.NewConfig().WithRegion(string(region)))
```

Comment:
This change is not related to logging. Would you mind removing this change (put it into a separate PR please)